src/schema/objects/metadata.yaml

```diff
@@ -882,6 +921,43 @@ EchoTime__fmap:
   type: number
   unit: s
   exclusiveMinimum: 0
+EditCondition:
```

Comment:
Are there any other fields where EditCondition, etc may be necessary in the future in a different context? Otherwise this could be named `SpectralEditCondition`, but if you don't forsee any, good to keep it like this.

Reply:
"Editing" is equivalent to "spectral editing", which is used for fat suppression in MRI. So this might need to be harmonized with the `WaterSuppression` and `OuterVolumeSuppression` metadata fields we've already proposed.

Reply:
DICOM doesn't have a specific tag for spectral editing, so I think what we have is suitable.